css-two-column-layout/index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Two Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <!-- entire container -->
+  <div class="container">
+    <!-- row one, which is the header and the paragraph under it -->
+    <div class="row">
+      <div class="header-content">
+        <h1 class="header-content-h1">SCALE, SPEED, AGILITY</h1>
+        <p class="header-content-p">We've dedicated thousands of man hours to making healthcare institutions in
+          the Helium network run 10x more efficiently and deliver an unparalleled quality
+          of care.
+        </p>
+
+      </div>
+    </div>
+    <!-- second row, this will contain the images, headers and paragraphs -->
+    <div class="row">
+      <div class="content-section-div">
+        <!-- first row within content section -->
+        <div class="content-section-row">
```

Comment:
What is the purpose of these two elements and their classes? This can just be one `row` (that is able to wrap) with four `column-half` elements inside of it, one for each section. You shouldn't need contextually specific `row` or `column` classes, they are meant to be infinitely reusable. If you need a `row` or `column` element with more styling than just what is covered by the layout class, you can make an additional class to add **_in addition to_** the regular `row` or `column` class

Reply:
Just made the changes. Thank you for the feedback. Having 4 divs in the content-div class makes more sense and makes the code look cleaner.

Reply:
<img width="903" alt="Screen Shot 2022-01-31 at 10 33 58 PM" src="https://user-images.githubusercontent.com/40590740/151923443-5f75f119-2625-4ca4-acba-6d93aed5b638.png">


Heres a screenshot of the changes if you need it.